common/djangoapps/course_modes/models.py

```diff
@@ -761,7 +761,10 @@ def min_course_price_for_currency(cls, course_id, currency):
         If there is no mode found, will return the price of DEFAULT_MODE, which is 0
         """
         modes = cls.modes_for_course(course_id)
-        return min(mode.min_price for mode in modes if mode.currency.lower() == currency.lower())
+        return min(
+            (mode.min_price for mode in modes if mode.currency.lower() == currency.lower()),
+            default=0
```

Comment:
Is there a way to get this default from the `DEFAULT_MODE`, rather than hardcoding a magic value?